zerver/lib/bugdown/__init__.py

```diff
@@ -1835,17 +1879,18 @@ def build_inlinepatterns(self) -> markdown.util.Registry:
         # rules, that preserves the order from upstream but leaves
         # space for us to add our own.
         reg = markdown.util.Registry()
-        reg.register(BacktickPattern(BACKTICK_RE), 'backtick', 105)
-        reg.register(markdown.inlinepatterns.DoubleTagPattern(STRONG_EM_RE, 'strong,em'), 'strong_em', 100)
-        reg.register(UserMentionPattern(mention.find_mentions, self), 'usermention', 95)
-        reg.register(Tex(r'\B(?<!\$)\$\$(?P<body>[^\n_$](\\\$|[^$\n])*)\$\$(?!\$)\B'), 'tex', 90)
-        reg.register(StreamPattern(get_compiled_stream_link_regex(), self), 'stream', 85)
-        reg.register(Avatar(AVATAR_REGEX, self), 'avatar', 80)
-        reg.register(ModalLink(r'!modal_link\((?P<relative_url>[^)]*), (?P<text>[^)]*)\)'), 'modal_link', 75)
+        reg.register(BacktickPattern(BACKTICK_RE), 'backtick', 110)
+        reg.register(markdown.inlinepatterns.DoubleTagPattern(STRONG_EM_RE, 'strong,em'), 'strong_em', 105)
+        reg.register(UserMentionPattern(mention.find_mentions, self), 'usermention', 100)
+        reg.register(Tex(r'\B(?<!\$)\$\$(?P<body>[^\n_$](\\\$|[^$\n])*)\$\$(?!\$)\B'), 'tex', 95)
+        reg.register(StreamPattern(get_compiled_stream_link_regex(), self), 'stream', 90)
+        reg.register(Avatar(AVATAR_REGEX, self), 'avatar', 85)
+        reg.register(ModalLink(r'!modal_link\((?P<relative_url>[^)]*), (?P<text>[^)]*)\)'), 'modal_link', 80)
         # Note that !gravatar syntax should be deprecated long term.
-        reg.register(Avatar(GRAVATAR_REGEX, self), 'gravatar', 70)
-        reg.register(UserGroupMentionPattern(mention.user_group_mentions, self), 'usergroupmention', 65)
-        reg.register(AtomicLinkPattern(get_link_re(), self), 'link', 60)
+        reg.register(Avatar(GRAVATAR_REGEX, self), 'gravatar', 75)
+        reg.register(UserGroupMentionPattern(mention.user_group_mentions, self), 'usergroupmention', 70)
+        reg.register(AtomicLinkPattern(get_link_re(), self), 'link', 65)
+        reg.register(GithubLink(get_compiled_github_link_regex(), self), 'github_link', 60)
```

Comment:
I think it'd be reasonable for this feature to just by 56 rather than renumbering everything up by 5, given that structurally we want it to be right after 55.

Reply:
Sure. I thought there was a reason for increasing them by 5.

Reply:
Yeah, the reason was the leave space for things inserted in between later, like this :)